management/organization.go

```diff
@@ -23,6 +23,11 @@ type Organization struct {
 	// Metadata associated with the organization, in the form of an object with
 	// string values (max 255 chars). Maximum of 10 metadata properties allowed.
 	Metadata *map[string]string `json:"metadata,omitempty"`
+
+	// OrganizationConnections that will be enabled for this organization.
+	// See POST enabled_connections endpoint for the object format.
+	// (Max of 10 connections allowed)
+	OrganizationConnections []*OrganizationConnection `json:"enabled_connections,omitempty"`
```

Comment:
I think we should maintain the naming of the json property unless we have a clear reason to not do so
```suggestion
	EnabledConnections []*OrganizationConnection `json:"enabled_connections,omitempty"`
```